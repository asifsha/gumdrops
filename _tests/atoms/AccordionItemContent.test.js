/* globals mount */
import React from 'react';
import AccordionItemContent from '../../components/atoms/AccordionItemContent';

describe('Expect <AccordionItemContent>', () => {
    it('to render', () => {
        const props = {
            className: 'foo',
            size: 'xs',
            context: 'danger'
        };
        const wrapper = mount(
            <AccordionItemContent {...props}>
                <div>foo</div>
            </AccordionItemContent>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
