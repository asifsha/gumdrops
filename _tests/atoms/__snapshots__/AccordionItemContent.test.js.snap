// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <AccordionItemContent> to render 1`] = `
<AccordionItemContent
  className="foo"
  context="danger"
  size="xs"
>
  <li
    className="gds-accordion__child-item foo gds-accordion__child-item--danger"
    onClick={[Function]}
  >
    <h4
      className="gds-accordion__child-item-title gds-accordion__child-item-title--xs"
    >
      <div>
        foo
      </div>
    </h4>
  </li>
</AccordionItemContent>
`;
